Week-14/Link_List_1/Reverse_every_k.js

```diff
@@ -0,0 +1,16 @@
+function reverseEveryK(head, k) {
```

Comment:
[mandatory]: declare the linkedlist class

Reply:
declared